### PR TITLE
Add Schema for Managed Application ViewDefinition 

### PR DESCRIPTION
**Pull request by jjbfour**
Add Schema for Managed Application ViewDefinition

### CODE REVIEW

File: schemas/viewdefinition/0.0.1-preview/ViewDefinition.json [hunk 1]
```diff
@@ -0,0 +1,399 @@
...
+        "contentVersion",
+        "views"
+    ],
+    "definitions": {
+        "View": {
+            "discriminator": "kind",
+            "properties": {
+                "kind": {
+                    "type": "string",
+                    "description": "The kind of the view.",
```
Comment (bmoore-msft) [inline schemas/viewdefinition/0.0.1-preview/ViewDefinition.json]: s/b - "The kind of view."

File: schemas/viewdefinition/0.0.1-preview/ViewDefinition.json [hunk 2]
```diff
@@ -0,0 +1,399 @@
...
+                    },
+                    "enum": [
+                        "Overview",
+                        "Metrics",
+                        "CustomResources"
+                    ]
+                },
+                "name": {
+                    "type": "string",
+                    "description": "The name of the view"
```
Comment (bmoore-msft) [inline schemas/viewdefinition/0.0.1-preview/ViewDefinition.json]: we should be consistent on the punctuation for the descriptions (either always add or never add) - I don't think we have a standard so would default to including it, given the use cases

File: schemas/viewdefinition/0.0.1-preview/ViewDefinition.json [hunk 3]
```diff
@@ -0,0 +1,399 @@
...
+        "CustomResources": {
+            "allOf": [
+                {
+                    "$ref": "#/definitions/View"
+                }
+            ],
+            "properties": {
+                "kind": {
+                    "type": "string",
+                    "description": "The kind of the view.",
```
Comment (bmoore-msft) [inline schemas/viewdefinition/0.0.1-preview/ViewDefinition.json]: The kind of view

File: schemas/viewdefinition/0.0.1-preview/ViewDefinition.json (hunk 1 above)
Comment (jjbfour) [reply schemas/viewdefinition/0.0.1-preview/ViewDefinition.json]: updated

File: schemas/viewdefinition/0.0.1-preview/ViewDefinition.json (hunk 3 above)
Comment (jjbfour) [reply schemas/viewdefinition/0.0.1-preview/ViewDefinition.json]: updated

File: schemas/viewdefinition/0.0.1-preview/ViewDefinition.json (hunk 2 above)
Comment (jjbfour) [reply schemas/viewdefinition/0.0.1-preview/ViewDefinition.json]: updated to add punctuation.

**Comment by Tiano2017** [comment]
@bmoore-msft is there any issue remaining on this PR?

**Comment by bmoore-msft** [comment]
LGTM

**Pull request merged**